src/components/SubjectPhasePicker/SubjectPhasePicker.test.tsx

```diff
@@ -108,9 +112,9 @@ describe("Component - subject phase picker", () => {
       <SubjectPhasePicker {...subjectPhaseOptions} />,
     );
     await userEvent.click(getByTitle("Subject"));
-    expect(queryByText("Latest resources")).toBeTruthy();
+    expect(queryByText("New curriculum plans")).toBeTruthy();
```

Comment:
Sorry about these bad tests :/ I wrote them before I knew to use `data-testid` instead of looking for specific text. You can update them if you're up for it, otherwise I'm happy to handle it in a separate ticket.